compiler/src/main/scala/play/twirl/compiler/TwirlCompiler.scala

```diff
@@ -732,7 +733,30 @@ package """ :+ packageName :+ """
               }
               .mkString(",") + ")"
           )
+          .zipWithIndex
+          .map { case (groupStr, idx) =>
+            // Finds last element and prepends "using" if the definition site has implicit parameter eg - apply(x)(y) becomes apply(x)(using y)
+            if (TwirlCompiler.isScala3 && hasContextParameters && idx == params.size - 1)
+              groupStr.replace("(", "(using ")
+            else groupStr
+          }
```

Comment:
This seems to be working well.
However, I thought it might be more readable if the processing were completed within the previous map { ... } block, rather than using the `zipWithIndex.map`.

Reply:
I thought it wouldn’t be necessary to check whether a parameter is the last, since an implicit parameter is always the last parameter group.
However, I think your current implementation is fine. thanks.